src/main/java/com/pokegoapi/api/map/Map.java

```diff
@@ -244,6 +244,7 @@ public FortDetails getFortDetails(String id, long lon, long lat) throws LoginFai
 		return new FortDetails(response);
 	}
 
+    @Deprecated
```

Comment:
Please fix this indentation, @Grover-c13 prefers tabs and I'd like to keep everything consistent.